### PR TITLE
[site] Added pprof doc

### LINKED ISSUE #983: Include the documentation for using pprof
Add instructions on how to connect to pprof
ref: https://github.com/kubernetes-sigs/kueue/pull/978#discussion_r1262632303

/kind documentation

### PR DESCRIPTION
**Pull request by stuton**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
[site] Added pprof doc

#### Which issue(s) this PR fixes:

Fixes #983

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/installation/_index.md [hunk 1]
```diff
@@ -205,3 +206,26 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+
+## Enabling pprof endpoints
+
+> _Available in Kueue v0.4.0 and later_
```
Comment (trasc) [inline site/content/en/docs/installation/_index.md:212]:
```suggestion
> _Available in Kueue v0.5.0 and later_
```

File: site/content/en/docs/installation/_index.md [hunk 2]
```diff
@@ -205,3 +206,26 @@ To uninstall Kueue, run the following command:
...
+Forwarding from 127.0.0.1:8082 -> 8082
+Forwarding from [::1]:8082 -> 8082
+```
+
+The HTTP endpoint will now be available as a local port.
+For example, you can now generate the file for the CPU profile with curl and pipe the data to a file (600 seconds is 10 minutes)
+
+```shell
+curl "http://localhost:8082/debug/pprof/profile?seconds=600" > cpu.pprof
+```
```
Comment (trasc) [inline site/content/en/docs/installation/_index.md:229]: Maybe just add another reftrence to https://pkg.go.dev/net/http/pprof#hdr-Usage_examples for how to use the endpoints.

**Review by trasc — COMMENTED** [review]
We should target `website-release-0.5` branch with this

**Comment by stuton** [comment]
duplicate #1003

**Pull request closed without merging**